qemu/tests/cfg/rng_bat.cfg

```diff
@@ -24,3 +24,5 @@
         driver_verifier_cmd = "cat /sys/devices/virtual/misc/hw_random/rng_current"
         driver_name = virtio
         rng_data_rex = "\w+"
+        check_rngd_service = "service rngd status"
+        start_rngd_service = "service rngd start && rngd -r /dev/hwrng -o /dev/random"
```

Comment:
service cmd is not working well on rhel8.

Reply:
'service' command works for me:
RHEL 7: 3.10.0-933.el7.x86_64
RHEL 8: 4.18.0-2.el8.x86_64
I use it mainly for compatibility with RHEL 6.
Thanks!

Reply:
RHEL7 and RHEL8 recommend systemctl to control system services.  But If service rngd start/stop/status command still work for 6&7&8.  I think it is enough for this case.